React.podspec

```diff
@@ -1,140 +1,138 @@
-                            There is no browser and no HTML. We have picked what
-                            we think is the best set of features from these and
-                            other technologies to build what we hope to become
-                            the best product development framework available,
-                            with an emphasis on iteration speed, developer
-                            delight, continuity of technology, and absolutely
-                            beautiful and fast products with no compromises in
-                            quality or capability.
-                         DESC
-  s.homepage            = "http://facebook.github.io/react-native/"
-  s.license             = package['license']
-  s.author              = "Facebook"
-  s.source              = { :git => "https://github.com/facebook/react-native.git", :tag => "v#{s.version}" }
-  s.default_subspec     = 'Core'
-  s.requires_arc        = true
-  s.platform            = :ios, "8.0"
-  s.pod_target_xcconfig = { "CLANG_CXX_LANGUAGE_STANDARD" => "c++14" }
-  s.header_dir          = 'React'
-  s.preserve_paths      = "package.json", "LICENSE", "LICENSE-CustomComponents", "PATENTS"
+  s.name                    = "React"
+  s.version                 = package["version"]
+  s.summary                 = package["description"]
+  s.description             = <<-DESC
+                                React Native apps are built using the React JS
+                                framework, and render directly to native UIKit
+                                elements using a fully asynchronous architecture.
+                                There is no browser and no HTML. We have picked what
+                                we think is the best set of features from these and
+                                other technologies to build what we hope to become
+                                the best product development framework available,
+                                with an emphasis on iteration speed, developer
+                                delight, continuity of technology, and absolutely
+                                beautiful and fast products with no compromises in
+                                quality or capability.
+                             DESC
+  s.homepage                = "http://facebook.github.io/react-native/"
+  s.license                 = package["license"]
+  s.author                  = "Facebook"
+  s.source                  = { :git => "https://github.com/facebook/react-native.git", :tag => "v#{s.version}" }
+  s.default_subspec         = "Core"
+  s.requires_arc            = true
+  s.platform                = :ios, "8.0"
+  s.pod_target_xcconfig     = { "CLANG_CXX_LANGUAGE_STANDARD" => "c++14" }
+  s.preserve_paths          = "package.json", "LICENSE", "LICENSE-CustomComponents", "PATENTS"
+  s.cocoapods_version       = ">= 1.2.0.beta.1"
```

Comment:
This version of the spec uses `POD_TARGET_SRCROOT`, which is a CocoaPods 1.2.0 only feature. I'll have another think when finalizing it.